src/mpi/datatype/datatype_impl.c

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) by Argonne National Laboratory
+ * Copyright (c) 2020      IBM Corporation.  All rights reserved.
```

Comment:
External copyright contributions is noted in the top level COPYRIGHT file. We no longer note additional copyright in individual files.

Reply:
Thanks, I just now removed them